export const GENERIC_ERROR_MESSAGE =
    "Something unexpected happened. Please try again later."
